pure_changelog.md

```diff
@@ -0,0 +1,194 @@
+* Fixed problem with keyboard buttons hanging up.
+
+## [1.2.1 2022-06-10]
+
+### Improved
+* Improved audio quality
+
+## [1.2.0 2022-04-01]
+
+### Added
+
+#### Quotes:
+* Possibility to set periodically drawn quotes as wallpaper - predefined and custom.
+
+#### Messages:
+* Text writing option - "Abc" mode.
+
+#### Contacts:
+* Functionality to support the display of long contact names.
+
+#### General:
+* Extended time for displaying instructions to unlock the device.
+
+#### Music Player:
+* Extended audio files format support.
+
+### Changed / Improved
+
+#### General:
+* Changed the default text input type from ABC mode to Abc mode.
+
+### Fixed
+
+* Fixed problem with missing gong at beginning and end of meditation process.
+* Fixed issue with setting a time of 0 minutes for meditation.
+* Fixed issue with the inability to delete characters when entering the meditation duration.
+* Fixed problem with missing interval gong in default settings in Meditation.
+* Fixed problem with receiving messages in offline mode.
+* Fixed problem with previewing longer messages as they are entered.
+* Fixed problem with restarting a fully discharged Pure.
+* Fixed a problem with navigating through longer lists available in the system.
+* Fixed problem with no sound for incoming calls/messages.
+* Fixed problem with keyboard stuttering.
+* Fixed problem with the phone hanging up after connecting/disconnecting the USB cable.
+* Fixed problem with alarm turning off when Pure is locked.
+* Fixed problem with alarm going off during active snooze time.
+* Fixed problem with incorrect display of time on Home Screen.
+* Fixed problem with leaving emergency call dialing screen on locked screen.
+* Fixed problem with keyboard unlock instructions disappearing too quickly.
+* Fixed an issue with unread messages/missed calls appearing on the lock screen when opened.
+* Fixed problem with displaying the list of available languages in non-alphabetical order.
+* Fixed problem with wrong order of items - date and time editing screen.
+* Fixed issue with text in other languages being cut off too long in Contacts.
+* Fixed problem with editing/adding numbers when there is a dash in the phone number.
+* Fixed problem with alphabetical sorting of contacts that do not contain the last name.
+* Fixed problem of being able to edit a deleted contact.
+* Fixed issue with the text informing about the Mudita OS update that appears during onboarding.
+* Fixed problem with using call log when more than 900 accounts are loaded.
+* Fixed problem with missing call information from the private number in the notification.
+* Fixed issue with the inability to interrupt an incoming quick call.
+* Fixed translations when adding, editing and deleting contacts from the call log.
+* Fixed issue with restoring data on Pure using Mudita Center.
+* Fixed issue with FLAC file playback.
+* Fixed problem with Music Player application crashing when appearing on file with unknown formats.
```

Comment:
"Fixed problem with Music Player application crashing when trying to play file with unknown format." sounds better IMHO ;)

Reply:
fixed